akka-http-core/src/main/scala/akka/http/impl/engine/http2/Http2Multiplexer.scala

```diff
@@ -77,12 +77,10 @@ private[http2] trait Http2MultiplexerSupport { logic: GraphStageLogic with Stage
   /** Called by the multiplexer after SETTINGS_INITIAL_WINDOW_SIZE has changed */
   def distributeWindowDeltaToAllStreams(delta: Int): Unit
 
-  def createMultiplexer(outlet: GenericOutlet[FrameEvent], prioritizer: StreamPrioritizer): Http2Multiplexer =
+  def createMultiplexer(pushFrameOut: FrameEvent => Unit, prioritizer: StreamPrioritizer): Http2Multiplexer with OutHandler =
```

Comment:
You can just define `pushFrameOut` above as we did for other callbacks back to `Http2MultiplexerSupport`.